### DISCUSSION TITLE
Indicator background color

### DISCUSSION THREAD

**Discussion by alexandrefelipemuller** [AppIntro/AppIntro#989] [General]
Hi, thanks for your help on other compiling issue.
I could add my AppIntro Activity and call it, but for default Slides and Indicator are white background with letter in white, and the buttons and other stuffs too. Is that a way to set indicator background color? is that something missing to declare or import for colors?

**Comment by cortinico** [comment]
> I could add my AppIntro Activity and call it, but for default Slides and Indicator are white background with letter in white, and the buttons and other stuffs too.

Could you provide a screenshot?

**Comment by alexandrefelipemuller** [comment]
This is the normal one:
![image](https://user-images.githubusercontent.com/15862398/147781518-77f0a350-477c-4aad-b0b8-f8535e19a71b.png)
This is the one I forced Title description and backcolor
![image](https://user-images.githubusercontent.com/15862398/147781592-4041e3b4-7135-4903-966f-164c13b0c480.png)

**Reply by cortinico** [reply]
Ok so the problem here is because you're probably editing the default theme/styles. The AppIntro styles are defined here: https://github.com/AppIntro/AppIntro/blob/6c8ad93fe763ea09c51862aa729b99650a8632ef/appintro/src/main/res/values/styles.xml#L4-L56

You can override them by providing styles with the same name and apply the customization you wish. Like:

```
    <style name="AppIntroDefaultText" parent="TextAppearance.AppCompat.Body1">
        <item name="android:textColor">...your custom color here...</item>
    </style>
```

**Reply by alexandrefelipemuller** [reply]
Thanks for your promptly response, the strange thing that I have no custom styles defined in my project, when I add this style it shows the slides text, but not the indicator (specially buttons), I've tried to override it but the indicator seems not to take effect

**Reply by cortinico** [reply]
Would you be able to provide a reproducer?

**Comment by alexandrefelipemuller** [comment]
sure,
My project: https://github.com/alexandrefelipemuller/plantatreeapp.git

onCreateView of HomeFragment.kt remove comments for the code:
```
        var intent = Intent(this.context, Tutorial::class.java)
        startActivity(intent)
```

**Comment by cortinico** [comment]
I've just tested your project and this is what I see.
I'm unsure what is the problem in this screenshot.
![image](https://user-images.githubusercontent.com/3001957/148655093-c5eb2523-ce75-41d9-be55-dec85bce14d6.png)

**Comment by alexandrefelipemuller** [comment]
That's interesting, Are you using DarkMode? I installed on a Samsung Galaxy S8, and when I use normal day mode it looks like screenshot, but when I switch dark mode it show like you screenshot

**Reply by cortinico** [reply]
yes I'm on dark mode 👍

**Reply by ankandrew** [reply]
I also get the same results (as @alexandrefelipemuller screenshot) when using Day mode. How can get around this, to make it work for both Day and Night modes?

**Reply by cortinico** [reply]
Without a reproducer I can't really debug/fix this.
